src/axom/primal/geometry/NURBSPatch.hpp

```diff
@@ -2324,7 +2518,7 @@ class NURBSPatch
   }
 
   /*!
-   * \brief Splits a NURBS surface (at each internal knot) into several Bezier patches
+   * \brief Splits the untrimmed NURBS surface (at each internal knot) into several Bezier patches
```

Comment:
Please consider mentioning something about how this function deals with trimming curves